R/g.report.part4.R

```diff
@@ -69,7 +69,13 @@ g.report.part4 = function(datadir = c(), metadatadir = c(), loglocation = c(), f
       if (length(dir(paste(metadatadir, ms5.out, sep = ""))) == 0) {
         try.add.part5.variable = FALSE  #do not run this function if there is no milestone data from g.part5
       } else {
-        try.add.part5.variable = TRUE
+        # check WW windows are calculated in the first file
+        load(dir(paste(metadatadir, ms5.out, sep = ""), full.names = T)[1])
+        if ("WW" %in% output$window) { 
```

Comment:
Is this check necessary? Isn't nonwear_perc_spt always calculated in part 5?

Reply:
This checks if there are "WW" windows in the part 5 output data, I think we need this to decide if nonwear_perc_spt will be added to part 4 output data or not.